### PR TITLE
Update eg.m3u

### PR DESCRIPTION
**Pull request by Carlinhos027**

### CODE REVIEW

File: streams/eg.m3u [hunk 1]
```diff
@@ -1,25 +1,19 @@
 #EXTM3U
 #EXTINF:-1 tvg-id="AghapyTV.eg",Aghapy TV (1080p) [Not 24/7]
 https://5b622f07944df.streamlock.net/aghapy.tv/aghapy.smil/playlist.m3u8
-#EXTINF:-1 tvg-id="AlfathSonnahTV.eg",Alfath Sonnah TV (576p)
-https://alfat7-q.com:5443/LiveApp/streams/713332814575062108399616.m3u8
```
Comment (UltraHDR) [inline streams/eg.m3u:5]: https://alfat7-q.com:5443/LiveApp/streams/515112551489055772942045.m3u8

Comment (UltraHDR) [reply streams/eg.m3u:5]: Alfath kids
https://alfat7-q.com:5443/LiveApp/streams/434833549141598440488987.m3u8

File: streams/eg.m3u [hunk 2]
```diff
@@ -1,25 +1,19 @@
...
-#EXTINF:-1 tvg-id="AlfathTV.eg",Alfath TV (576p)
-https://alfat7-q.com:5443/LiveApp/streams/117655467302550619118331.m3u8
```
Comment (UltraHDR) [inline streams/eg.m3u:7]: https://alfat7-q.com:5443/LiveApp/streams/515112551489055772942045.m3u8

File: streams/eg.m3u [hunk 3]
```diff
@@ -1,25 +1,23 @@
 #EXTM3U
 #EXTINF:-1 tvg-id="AghapyTV.eg",Aghapy TV (1080p) [Not 24/7]
 https://5b622f07944df.streamlock.net/aghapy.tv/aghapy.smil/playlist.m3u8
-#EXTINF:-1 tvg-id="AlfathSonnahTV.eg",Alfath Sonnah TV (576p)
-https://alfat7-q.com:5443/LiveApp/streams/713332814575062108399616.m3u8
 #EXTINF:-1 tvg-id="AlfathTV.eg",Alfath TV (576p)
-https://alfat7-q.com:5443/LiveApp/streams/117655467302550619118331.m3u8
+https://alfat7-q.com:5443/LiveApp/streams/515112551489055772942045.m3u8
+#EXTINF:-1 tvg-id="AlfathSonnahTV.eg",Alfath Sonnah TV (576p)
+https://alfat7-q.com:5443/LiveApp/streams/434833549141598440488987.m3u8
```
Comment (UltraHDR) [inline streams/eg.m3u:7]:
```suggestion
https://alfat7-q.com:5443/LiveApp/streams/434833549141598440488987.m3u8
#EXTINF:-1 tvg-id="",Alfath Kids TV (576p)
https://alfat7-q.com:5443/LiveApp/streams/434833549141598440488987.m3u8
```

**Review by UltraHDR — APPROVED** [review]

**Review by freearhey — APPROVED** [review]

**Pull request merged**